src/mixins/amalgamation-mixin.ts

```diff
@@ -320,22 +321,34 @@ export default class AmalgamationMixin extends Vue {
           foreignJurisdiction: item.foreignJurisdiction
         } as AmalgamatingBusinessIF
       } else {
-        const tingBusiness = await this.fetchAmalgamatingBusinessInfo(item.identifier)
+        const business = await this.fetchAmalgamatingBusinessInfo(item.identifier)
+
+        // if auth info is empty then business is not (no longer) affiliated with current account
+        if (!business.authInfo) {
+          return {
+            type: AmlTypes.LEAR,
+            role: AmlRoles.AMALGAMATING,
+            identifier: item.identifier,
+            name: item.name,
+            legalType: item.legalType
+          } as AmalgamatingBusinessIF
+        }
```

Comment:
This item will appear as "not affiliated".

It should never happen that the user was able to add the business but now it doesn't exist at all.